package.json

```diff
@@ -4,9 +4,10 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "test": "echo \"Currently No Tests\"",
-    "start": "npm run build && node dist/app.js",
-    "build": "npx tsc && cp -R views dist && cp -R static dist"
+    "test": "mocha dist/test/unit/**/*.js",
```

Comment:
There is a better way to run the tests, instead of putting them in dist (which we shouldn't need to do as dist should be app code only)

`mocha --require ts-node/register test/unit/**/*.ts`

You'll need to install ts-node as a dev dependency and also update the tsconfig file to not put the test files in the dist folder

Reply:
You may find sorting out the folder structure helps with the above (application code in stc folder)

Reply:
- Moved `app.ts` and `router.ts` into `src`
- Added `ts-node` dependency
- Updated `tsconfig.json` to only use `src`
- Updated test command in `package.json`